src/index.test.js

```diff
@@ -324,3 +324,57 @@ describe('time zone is carried over after a business-day operation', () => {
     expect(nyPlusTwo.zoneName).toBe('America/New_York');
   });
 });
+
+describe('diffBusiness()', () => {
+  it('knows two identical DateTimes have a business day diff of 0', () => {
+    const targetDate = DateTime.local().plus({ hours: 2 });
```

Comment:
```suggestion
    const targetDate = DateTime.local().startOf('day').plus({ hours: 2 });
```

Reply:
I committed your suggestion, feel free to resolve at your convenience.